### PR TITLE
fix: Always get the same value where you want to get it at random.

### PR DESCRIPTION
**Pull request by soranoba**
```
phone.go:70:47: SA4030: (*math/rand.Rand).Intn(n) generates a random value 0 <= x < n; that is, the generated values don't include n; rand.Intn(1) therefore always returns 0 (staticcheck)
	return fmt.Sprintf("(%s) %s", boxDigitsStart[rand.Intn(1)], out)
	                                             ^
phone.go:94:45: SA4030: (*math/rand.Rand).Intn(n) generates a random value 0 <= x < n; that is, the generated values don't include n; rand.Intn(1) therefore always returns 0 (staticcheck)
	return fmt.Sprintf("+%s%s", boxDigitsStart[rand.Intn(1)], strings.Join(slice.IntToString(ints), ""))
	                                           ^
```

### CODE REVIEW

**Comment by bxcodec** [comment]
Hi @soranoba 

Can you re-fixing this again, I just fixing some linter issue in #167

File: phone.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ func (p Phone) tollfreephonenumber() string {
 		}
 		out += v
 	}
-	return fmt.Sprintf("(%s) %s", boxDigitsStart[rand.Intn(1)], out)
+	return fmt.Sprintf("(%s) %s", boxDigitsStart[rand.Intn(len(boxDigitsStart))], out)
```
Comment (soranoba) [inline phone.go:70]: `rand.Intn(1)` always returns 0.
So, `boxDigitsStart[rand.Intn(1)]` always is "777".

This fix is to correctly return random values.

**Comment by soranoba** [comment]
@bxcodec Thank you. I did it.

**Review by bxcodec — APPROVED** [review]
LGTM

**Comment by bxcodec** [comment]
Thanks @soranoba merged

**Pull request merged**